core/src/models/App.ts

```diff
@@ -136,7 +133,7 @@ export class App extends LoggedModel<App> {
     return OptionHelper.getPlugin(this);
   }
 
-  async setConnection(connection) {
+  async setConnection(connection: any) {
```

Comment:
This one is tricky though... it could be everything from an AxiosClient to a BigQueryCustomConnectionThing... and those types don't exist in `@grouparoo/core`... they are from the plugin.  Not really sure what to do here

Reply:
I really don't think typing this will be helpful.  We could make it variable, eg:
<img width="619" alt="Screen Shot 2022-01-10 at 10 36 13 AM" src="https://user-images.githubusercontent.com/303226/148820384-ad1ab0b5-51a8-4bfb-ace9-1d3adcf35741.png">

but again, no one in core has the type to provide, it's only going to be defined in the plugins... so all uses of these methods in this core project will always be passing undefined or any regardless